openedx_events/event_bus/avro/tests/test_avro.py

```diff
@@ -11,10 +17,58 @@
     deserialize_bytes_to_event_data,
     serialize_event_data_to_bytes,
 )
+from openedx_events.learning import signals  # pylint: disable=reimported
+from openedx_events.tooling import OpenEdxPublicSignal
+
+KNOWN_UNSERIALIZABLE_EVENTS = ["org.openedx.learning.discussions.configuration.changed.v1"]
+
+def generate_test_event_data_for_signal(data_type):
```

Comment:
Also my bad, this should be `generate_test_data_for_data_type` or something since we're not actually passing it a signal